core/src/main/scala/com/karumi/shot/xml/ScreenshotsSuiteXmlParser.scala

```diff
@@ -54,11 +56,15 @@ object ScreenshotsSuiteXmlParser {
 
   def parseScreenshotSize(screenshot: Screenshot,
                           viewHierarchyContent: String): Screenshot = {
-    val xml = XML.loadString(viewHierarchyContent)
-    val screenshotWidth = (xml \ "right" head).text.toInt
-    val screenshotHeight = (xml \ "bottom" head).text.toInt
+    val json = parse(viewHierarchyContent)
+    val JInt(screenshotLeft) = json \ "left"
+    val JInt(screenshotWidth) = json \ "width"
+    val JInt(screenshotTop) = json \ "top"
+    val JInt(screenshotHeight) = json \ "height"
```

Comment:
Are the ``ScreenshotSuiteXmlParser`` tests passing after these changes? I see you are facing some issues with Travis and the emulator set up but we'd need to update the metadata file used in our tests before considering this PR as approved. [Here you have the test suit you should review.](https://github.com/Karumi/Shot/blob/master/core/src/test/scala/com/karumi/shot/xml/ScreenshotsSuiteXmlParserSpec.scala#L20)

Reply:
We have resolved it, this PR still under development, now, we have some issues with travis.